src/data/federal/federalPovertyGuidelines.js

```diff
@@ -1,6 +1,12 @@
+/** Data tables for Federal poverty guildelines/income limits.
+ *
+ * RESOURCES:
+ * {@link https://aspe.hhs.gov/poverty-guidelines}
+ *
+ * @namespace
+ */
 const federalPovertyGuidelines = { 0: 0, 1: 12060, 2: 16240, 3: 20420, 4: 24600, 5: 28780, 6: 32960, 7: 37140, 8: 41320, 'eachAdditional': 4180 };
+// {@link http://www.mass.gov/eohhs/docs/masshealth/deskguides/fpl-deskguide.pdf}
```

Comment:
Can this be put into some JSDoc block somewhere?

Reply:
I think not. Because it's a file name (with '.pdf' in its name), jsdoc seems to want to find that file in our folders and gets upset when it can't find it. It created errors when I tried to include that and then did `npm run build`. If that answers your questions, mark this comment as resolved.

Reply:
I could always have done something else wrong...